modules/virt-creating-a-vm-from-a-default-os-image.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-and-using-default-os-//images.adoc
+
+[id="virt-creating-a-vm-from-a-default-os-image_{context}"]
+= Creating a virtual machine from a default OS image
+
+After you have created a Persistent Volume Claim (PVC) for default OS images, you create a new virtual machine or virtual machine template from the PVC. Use the {product-title} Web Console wizard to create virtual machines or virtual machine templates.
+
+.Prerequisites
+
+* Select a bootable disk that you want to use as a default OS image.
+* You can create virtual machines and virtual machine templates with non-administrator privileges. You need administrator privileges to upload default OS images to PVCs.
+
+.Procedure
+
+. In the {product-title} Web Console, click *Workloads > Virtualization* in the sidebar menu. The *Virtualization* page displays. Use onscreen Help or existing documentation to create virtual machine templates.
+
+. Create a virtual machine or virtual machine template by specifying the PVC as the *Boot Source*.
++
+[NOTE]
+====
+This procedure describes how to create a virtual machine from a default OS image, uploaded using a PVC. The procedure to create a virtual machine template is identical to creating a virtual machine.
```

Comment:
"Using a PVC" is not correct. Not sure what did you want to say, but it is either "into a PVC" or "using the upload PVC form".

Reply:
Note has been deleted, content reworded and relocated.